core/tests/protractor_utils/editor.js

```diff
@@ -452,6 +466,28 @@ var _selectRule = function(ruleElement, interactionId, ruleName) {
   }
 };
 
+var selectRuleInAddResponse = function() {
+  var ruleElement = element(by.css('.protractor-test-add-response-details'));
+  var args = [ruleElement];
+  for (var i = 0; i < arguments.length; i++) {
+    args.push(arguments[i]);
+  }
+  _selectRule.apply(null, args);
+};
+
+var expectRuleParametersToBe = function() {
+  for (var i = 0; i < arguments.length; i++) {
+    var expected = arguments[i];
+    var parameterElement = element.all(
+      by.css('.protractor-test-answer-description-fragment'
+    )).get(i * 2 + 1).element(by.tagName('input')).getAttribute('value').then(
+      function(text) {
+        expect(text).toEqual(expected);
```

Comment:
@seanlip,
Yes, it doesn't work. 
In the new commit I used a different approach. 
It isn't perfect, but take a look.